libs/internal/console-driver/test-util/project.json

```diff
@@ -3,6 +3,22 @@
   "sourceRoot": "libs/internal/console-driver/test-util/src",
   "prefix": "ngworker",
   "targets": {
+    "build": {
```

Comment:
Why do we need the internal tools to be buildable? Are we not just using it for testing?

Reply:
See PR description and https://github.com/ngworker/lumberjack-next/pull/15#discussion_r891248700